python/dgl/data/wiki_network.py

```diff
@@ -0,0 +1,186 @@
+"""
+Wikipedia page-page networks on three topics: chameleons, crocodiles and
+squirrels
```

Comment:
I don't see crocodiles and squirrels related code.

Reply:
removed for now, will add later in another PR after we merge this one